akka-persistence-typed/src/main/scala/akka/persistence/typed/scaladsl/EventSourcedBehavior.scala

```diff
@@ -222,7 +222,7 @@ object EventSourcedBehavior {
    * Publish events to the system event stream as [[akka.persistence.typed.PublishedEvent]] after they have been persisted
    */
   @ApiMayChange
-  def withEventPublishing(): EventSourcedBehavior[Command, Event, State]
+  def withEventPublishing(enabled: Boolean): EventSourcedBehavior[Command, Event, State]
```

Comment:
I think this is more clear, and also makes it possible to toggle in both directions (in case we change the default to enable it by default)

Reply:
👍